README.md

```diff
@@ -56,6 +56,16 @@ get tricky, so please leave this directory out of your commits and pull request
 
 The committer doing the final merge will generate the `content/` directory at that time.
 
+### Running using Docker
+
+**Note:** This is an optional method if you don't want to install gems locally and just use a container for development.
```

Comment:
But you have a Bash script also? Running that on Windows is ~possible, but painful to the degree you'd probably want to install Ruby + gems instead. SGTM either way.